tests/test_keep_alive_timeout.py

```diff
@@ -101,46 +95,6 @@ def test_keep_alive_timeout_reuse():
             break
 
 
-@pytest.mark.skipif(
-    bool(environ.get("SANIC_NO_UVLOOP"))
-    or OS_IS_WINDOWS
-    or platform.system() != "Linux",
-    reason="Not testable with current client",
-)
-def test_keep_alive_client_timeout():
```

Comment:
We're axing this test completely. Is it just because we don't find the test useful or is there some other background?

Reply:
The test was hanging for a long time. It tests a timeout of httpx, which is essentially a disconnection on Sanic's side, and is not particularly meaningful to test.